cmd/cmd_update_client.go

```diff
@@ -42,6 +42,10 @@ To encrypt an auto-generated OAuth2 Client Secret, use flags ` + "`--pgp-key`" +
 			}
 
 			id := args[0]
+			if f := cmd.Flag(flagClientId); f.Changed {
```

Comment:
Can you please only add this flag to create_client?

Reply:
The `cmd_update_client.go` uses the same flags as `cmd_create_client.go`. Both use the flags defined in `cmd_helper_client.go`.